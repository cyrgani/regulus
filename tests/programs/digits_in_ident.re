=(1a1, 4),
=(22f2f2, 1a1),
assert_eq(22f2f2, 1a1),
def(123e, x, +(123, x)),
assert_eq(123e(16), 139),
__builtin_file_catch_assert_eq(
    assert_eq(16i64, 16)
)