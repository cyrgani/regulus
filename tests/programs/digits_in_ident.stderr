NameError: No variable named `16i64` found!
at programs/digits_in_ident.re:7:15
at programs/digits_in_ident.re:7:14
at programs/digits_in_ident.re:6:22
